tests/gpts/github/test-config/github_positive_suite.ts

```diff
@@ -195,7 +195,20 @@ export const gitHubBasicGoldenPathTemplateTests = (gptTemplate: string) => {
                 expect(finished).toBe(true)
             }
         }, 900000)
-
+  
+         /**
+         * Check if the pipelinerun yaml has the rh-syft image path mentioned
+         */
+         it(`Check ${gptTemplate} pipelinerun yaml has the rh-syft image path`, async ()=> {
+            const pipelineRun = await kubeClient.getPipelineRunByRepository(repositoryName, 'push')
+            if (pipelineRun && pipelineRun.metadata && pipelineRun.metadata.name) {
+                const doc = await kubeClient.pipelinerunfromName(pipelineRun.metadata.name,developmentNamespace)
+                const image_name = doc.spec.pipelineSpec.tasks[2].taskSpec.steps[1].image
```

Comment:
what if task change position to 1?

Reply:
Have added logic to search for task named"build-container" and search for image that has the string "registry.redhat.io/rh-syft-tech-preview/syft-rhel9 "